pixeltree/MyFirstBlogApi
# Posts

Adds post support. Includes a Post index view and individual Post views. 

And by views, as this is an API, I mean API response. 

No database or auth yet.

Testing: 
1. `dotnet run`
2. Go to `localhost:5001/swagger`. This exposes the endpoints that are available to the application. 
3. Get posts, try it out, click 'execute'
4. Copy the 'id' of a post
5. Get /posts/{id}, try it out, paste id, click execute
6. Ensure response body has desired info
